scripts/updatebom.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+if ! [[ "$0" =~ "scripts/updatebom.sh" ]]; then
+	echo "must be run from repository root"
+	exit 255
+fi
+
+echo "updating 'bill-of-materials.json'"
+go get -d -u github.com/coreos/license-bill-of-materials
+SHA_BOM="61fa1b0553b9fc2ddf8d292e03f405abdc3f6ac7"
```

Comment:
probably shouldn't hardcode the SHA since this might mask BOM bugs and CI uses master